Makefile

```diff
@@ -476,6 +482,7 @@ PUBLISH_AWS_S3 += \
 	$(BUILD_OUTPUT_DIRECTORY)/$(APPLICATION_NAME)-$(APPLICATION_VERSION)-$(TARGET_PLATFORM)-$(TARGET_ARCH).zip \
 	$(BUILD_OUTPUT_DIRECTORY)/$(APPLICATION_NAME)-$(APPLICATION_VERSION)-win32-$(TARGET_ARCH).exe \
 	$(BUILD_OUTPUT_DIRECTORY)/$(APPLICATION_NAME)-cli-$(APPLICATION_VERSION)-$(TARGET_PLATFORM)-$(TARGET_ARCH).zip
+installers-all: $(PUBLISH_AWS_S3)
```

Comment:
IIRC if the platform isn't Linux, then `$(PUBLISH_BINTRAY_DEBIAN)` will just be an empty variable, so you might be able to do `installers-all: $(PUBLISH_AWS_S3) $(PUBLISH_BINTRAY_DEBIAN)` just once (at the 'global' level), instead of having it inside multiple `ifeq` blocks?